.circleci/npmjs-publish.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+set -o pipefail
+export PACKAGE_VERSION=$(cat package.json \
+  | grep version \
+  | head -1 \
+  | awk -F: '{ print $2 }' \
+  | sed 's/[",]//g' \
+  | tr -d ' ')
+
+export PUBLISHED_VERSION=$(npm show @girder/components version \
+  | tr -d ' ')
```

Comment:
As advised in the blog @danlamanna linked, we ought to separate the `export` from the assignment, in case the shell would ignore failures from the right-hand command.

Reply:
Thanks I hadn't read that yet.  The export isn't needed at all, so I'll just drop it.